CHANGELOG.md

```diff
@@ -50,6 +50,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 ### New
 
+- **General**: Add parameter queryParameters in prometheus-scaler ([#4962](https://github.com/kedacore/keda/issues/4962))
```

Comment:
just a nit: move this to improvements please 🙏

Reply:
moved